scripts/algorithms/l2-svm-predict.dml

```diff
@@ -25,6 +25,19 @@
 # Given ground truth labels, the script will compute an 
 # accuracy (%) for the predictions
 #
+# INPUT PARAMETERS:
+# ---------------------------------------------------------------------------------------------
+# NAME      TYPE        DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# X         String      ---         Location to read the test feature matrix X
+# Y         String      ---         Location to read the true label matrix Y
```

Comment:
In the `MEANING`, we can simply add this `(Optional, and is needed only for evaluating performance(accuracy, confusion) of the model)` + No we can keep this here only.